keccak256/src/circuit/word_builder.rs

```diff
@@ -106,7 +106,7 @@ impl<F: Field> WordBuilderConfig<F> {
                     .iter()
                     .enumerate()
                     .map(|(idx, byte_cell)| {
-                        F::from(1u64 << (idx * 8)) * byte_cell.value().unwrap_or(&F::zero())
+                        F::from(1u64 << (idx * 8)) * byte_cell.value().cloned().unwrap_or_default()
```

Comment:
@CPerezz why do we copy elsewhere but clone here?

Reply:
My bad.. fixed it!!